src/platyPS/platyPS.psm1

```diff
@@ -599,6 +603,70 @@ function New-MarkdownAboutHelp
     }
 }
 
+function New-YamlHelp
+{
+    [CmdletBinding()]
+    [OutputType([System.IO.FileInfo[]])]
+    param(
+        [Parameter(Mandatory=$true,
+            Position=1,
+            ValueFromPipeline=$true,
+            ValueFromPipelineByPropertyName=$true)]
+        [string[]]$Path,
+
+        [Parameter(Mandatory=$true)]
+        [string]$OutputPath,
+
+        [System.Text.Encoding]$Encoding = [System.Text.Encoding]::UTF8,
+        
+        [switch]$Force
+    )
+	begin 
+	{
+		validateWorkingProvider
+
+		$MarkdownFiles = @()
+
+		if(-not (Test-Path $OutputPath))
+		{
+			mkdir $OutputPath -ErrorAction SilentlyContinue > $null
+		}
+
+		if(-not (Test-Path -PathType Container $OutputPath))
+		{
+			throw "$OutputPath is not a container"
+			return
+		}
+	}
+	process 
+	{
+		$MarkdownFiles += GetMarkdownFilesFromPath $Path
+	}
+	end 
+	{
+		$MarkdownFiles | ForEach-Object {
+            Write-Verbose "[New-ExternalHelp] Input markdown file $_"
+        }
+
+		foreach($markdownFile in $MarkdownFiles)
+		{
+			$mamlModels = GetMamlModelImpl $markdownFile.FullName -Encoding $Encoding
+			if($mamlModels.Count -ne 1)
```

Comment:
I've updated this block of code to handle a scenario where GetMamlModelImpl returns me more than one cmdlet